javascript/builtins/RegExp.json

```
@@ -1765,7 +1765,8 @@
                 "version_added": false
               },
               "nodejs": {
-                "version_added": true
+                "version_added": "6.0.0",
+                "notes": "Case folding is implemented in version 8.6.0"
```

Can we reframe this a little, to front-load the fact that it's a historic deficiency?

```suggestion
                "notes": "Before 8.6.0, case folding is not supported."
```

---

Good suggestion. I think I chose that wording to be consistent with the existing note for `edge` a few lines above. I'm happy to change it, but probably both notes should be changed together for consistency.

---

Ah, I missed that. Yeah, let's do that in a separate PR. I'll approve and merge this now. Thank you! 🎉